Stripe/STPBankSelectionTableViewCell.m

```diff
@@ -0,0 +1,115 @@
+@property (nonatomic, strong, readwrite) STPTheme *theme;
+
+@property (nonatomic, strong, readwrite) UIImageView *leftIcon;
+@property (nonatomic, strong, readwrite) UILabel *titleLabel;
+@property (nonatomic, strong, readwrite) UIActivityIndicatorView *activityIndicator;
+
+@end
+
+@implementation STPBankSelectionTableViewCell
+
+- (instancetype)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(nullable NSString *)reuseIdentifier {
+    self = [super initWithStyle:style reuseIdentifier:reuseIdentifier];
+    if (self) {
+        // Left icon
+        UIImageView *leftIcon = [[UIImageView alloc] init];
+        _leftIcon = leftIcon;
+        [self.contentView addSubview:leftIcon];
+
+        // Title label
+        UILabel *titleLabel = [UILabel new];
+        _titleLabel = titleLabel;
+        [self.contentView addSubview:titleLabel];
+
+        // Loading indicator
+        UIActivityIndicatorView *activityIndicator = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:UIActivityIndicatorViewStyleGray];
+        _activityIndicator = activityIndicator;
+        _activityIndicator.hidesWhenStopped = YES;
+        [self.contentView addSubview:activityIndicator];
+    }
+    return self;
+}
+
+- (void)layoutSubviews {
+    [super layoutSubviews];
+
+    CGFloat midY = CGRectGetMidY(self.bounds);
+    CGFloat padding = 15.0;
+    CGFloat iconWidth = 26.0;
+
+    // Left icon
+    [self.leftIcon sizeToFit];
+    self.leftIcon.center = CGPointMake(padding + (iconWidth / 2.0f), midY);
+
+    // Activity indicator
+    self.activityIndicator.center = CGPointMake(CGRectGetWidth(self.bounds) - padding - CGRectGetMidX(self.activityIndicator.bounds), midY);
+
+    // Title label
+    CGRect labelFrame = self.bounds;
+    // not every icon is `iconWidth` wide, but give them all the same amount of space:
+    labelFrame.origin.x = padding + iconWidth + padding;
+    labelFrame.size.width = CGRectGetMinX(self.activityIndicator.frame) - padding - labelFrame.origin.x;
+    self.titleLabel.frame = labelFrame;
+}
+
+- (void)configureWithBank:(STPFPXBankBrand)bankBrand theme:(STPTheme *)theme selected:(BOOL)selected enabled:(BOOL)enabled {
+    self.bank = bankBrand;
+    self.theme = theme;
+
+    self.backgroundColor = theme.secondaryBackgroundColor;
+
+    // Left icon
+    self.leftIcon.image = [STPImageLibrary brandImageForFPXBankBrand:self.bank];
+    self.leftIcon.tintColor = [self primaryColorForPaymentOptionWithSelected:selected enabled: enabled];
+
+    // Title label
+    self.titleLabel.font = theme.font;
+    self.titleLabel.text = STPStringFromFPXBankBrand(self.bank);
+    self.titleLabel.textColor = [self primaryColorForPaymentOptionWithSelected:self.selected enabled:enabled];
+
+    // Loading indicator
+    self.activityIndicator.tintColor = theme.accentColor;
+    if (selected) {
+        [self.activityIndicator startAnimating];
+    } else {
+        [self.activityIndicator stopAnimating];
```

Comment:
maybe a `bringSubviewToFront` as well in case the hierarchy here changes?

Reply:
er, on the start antimating I mean

Reply:
I don't think the hierarchy will change in these cells: The views are all added at initialization and never added/removed again. 
![Simulator Screen Shot - iPhone Xʀ - 2019-09-16 at 15 49 54](https://user-images.githubusercontent.com/52758633/64998610-b46aba00-d899-11e9-9756-f2c8458233ce.png)

Reply:
yeah I just mean more as a protection for in case someone comes in and changes the initializer without realizing. Not a blocker though